jazzy.sh

```diff
@@ -22,23 +22,25 @@ if [ -z "${GITHUB_USERNAME}" ] && [ -z "${GITHUB_PASSWORD}" ]; then
 fi
 
 # Check if .jazzy.yaml exists in the root folder of the repo
-#cd ..
-export projectFolder=`pwd`
+if [ ! -f "/.jazzy.yaml" ] then
+    echo ".jazzy.yaml file does not exist"
+    exit 1
+fi
 
-if [ -e ./$(projectFolder) ]; then
+# Checkout to the current branch
+git checkout "${TRAVIS_PULL_REQUEST_BRANCH}"
 
-    # Install jazzy
-    sudo gem install jazzy
-    # Generate xcode project
-    sourceScript "${projectFolder}/generate-xcodeproj.sh"
-    # Run jazzy
-    jazzy
+# Install jazzy
+sudo gem install jazzy
+# Generate xcode project
+sourceScript "/generate-xcodeproj.sh"
```

Comment:
Similar to the above, this is an absolute path to the root of the filesystem.  However in this case, you are referring to another script inside the Package-Builder project, not one within the project itself.

This is _probably_ going to be `$pwd/Package-Builder/generate-xcodeproj.sh` (and this is an assumption that `build-package.sh` currently makes about the location of `jazzy.sh`), however what happens if someone checks out Package-Builder to a different location, or with a different directory name?

What I'd suggest is defining a variable that represents the location of the current script, for example:
```bash
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
...
sourceScript "${SCRIPT_DIR}/generate-xcodeproj.sh"
```